lib/caches.js

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const LRU = require('lru-cache');
+
+// load config defaults
+require('./config');
+
+// module API
+const api = {};
+module.exports = api;
+
+bedrock.events.on('bedrock.init', async () => {
+  const cfg = bedrock.config.kms;
+  api.keyRecords = new LRU({max: cfg.keyRecordCache.maxSize});
```

Comment:
why is `cfg.keyRecordCache.maxAge` not used here as well.  I see that maxAge is being specified on the cache `set` calls, which is valid, but why?

Reply:
It was an oversight that you could pass in a default maxAge. I'll update it.

Reply:
Updated.